hawc/apps/assessment/api.py

```diff
@@ -170,6 +175,60 @@ def public(self, request):
         serializer = serializers.AssessmentSerializer(queryset, many=True)
         return Response(serializer.data)
 
+    @method_decorator(cache_page(60 * 60 * 24))
+    @action(detail=False, methods=("get",), renderer_classes=PandasRenderers)
+    def training_data(self, request):
```

Comment:
let's call this `bioassay-ml-dataset`, we'll likely have a different api call for epi